src/infra/modules/backend/lambda.tf

```diff
@@ -167,6 +167,25 @@ data "aws_iam_policy_document" "assertion_lambda" {
   }
 }
 
+module "security_group_lambda_assertion" {
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "4.17.2"
+
+  name        = "${var.assertion_lambda.name}-sg"
+  description = "Security Group for Lambda Assertion"
+
+  vpc_id = var.assertion_lambda.vpc_id
+
+  egress_cidr_blocks      = []
+  egress_ipv6_cidr_blocks = []
+
+  # Prefix list ids to use in all egress rules in this module
+  egress_prefix_list_ids  = [var.assertion_lambda.vpc_endpoint_events_prefix_id]
```

Comment:
mmmm this might be vice versa. I mean:
egress =>  dynamodb.
ingress => event bridge